config.go

```diff
@@ -141,3 +145,24 @@ func (c *Config) set(key string, value interface{}) {
 func (c *Config) Get(key string) interface{} {
 	return c.v.Get(key)
 }
+
+// NewFs is just for testing purposes.
+func NewFs(t *testing.T, configBytes []byte, configDir string, f afero.Fs) func() {
```

Comment:
It's a little weird that this takes both the config file bytes *and* a `afero.Fs`. I would expect it to just take the after.Fs and then have something else which generates the config file afterwards.

Or you could just take the config bytes and have the helper create the memory based Fs.